src/core/PlayHandle.cpp

```diff
@@ -60,6 +65,10 @@ void PlayHandle::doProcessing()
 
 void PlayHandle::releaseBuffer()
 {
-	BufferManager::release( m_playHandleBuffer );
```

Comment:
@lukas-w Why did you move this to destructor?

Reply:
@PhysSong Because I moved buffer allocation to the constructor. The buffer's lifetime matches the `PlayHandle`'s lifetime now, instead of being allocated everytime `doProcessing` is called.